pages.zh/common/aapt.md

```diff
@@ -1,16 +1,16 @@
 # aapt
 
-> 安卓资源包工具（Android Asset Packaging Tools）.
-> 该工具可以查看，创建， 更新资源压缩包(zip, jar, apk)。
+> 安卓资源包工具(Android Asset Packaging Tools）.
```

Comment:
Why didn't the right parentheses be replaced? `(）`

Reply:
Good question, I didn't really notice that. Should it even get replaced? Or does the same rule apply here (full-width braces around Chinese text, half-width braces around Latin text)?

Reply:
I just saw that this was already covered [here](https://github.com/tldr-pages/tldr/pull/5240#issuecomment-825281091). I'll update it.

Reply:
BTW, does all this only apply for `zh` or also for `zh_TW`? (@ChungZH)